### ISSUE TITLE
Exceptions handling inspection false-positive?

### ISSUE THREAD

**Issue by b1rdex** [kalessil/phpinspectionsea#739]
This is about `PHP | Php Inspections (EA Extended) | Control Flow | Exceptions handling and annotating`.
```php
class GoodSize
{
    public function setCount(int $count): void
    {
        if ($count < 0) {
            /* @noinspection ExceptionsAnnotatingAndHandlingInspection */
            throw new LogicException('Count must be greater than 0');
        }
        $this->count = $count;
    }
}

class Bar
{
    public function foo(): int
    {
        $size = new GoodSize();
        $size->setCount(\random_int(1, 15));
    }
}
```

`setCount` call in `Bar` class marked as throwing `LogicException` exception. But exception is muted with `@noinspection ExceptionsAnnotatingAndHandlingInspection`.
Is this correct behaviour or false positive?

**Comment by kalessil** [comment]
Nope, ` @noinspection` mutes the following statement only.
In this case, suppression makes sense near `$size->setCount(\random_int(1, 15));`

**Comment by b1rdex** [comment]
Hmm… looks weird for me. What muting that throw means than? I thought it's to make method signature as not throwing.

**Comment by kalessil** [comment]
It just suppresses the warning on this specific expression.
